lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -15500,6 +15500,9 @@ msgstr "Filtre id %s non trouvé"
 msgid "Filter already exists"
 msgstr "Le filtre existe déjà"
 
+msgid "Filter name already used"
```

Comment:
```suggestion
msgid "Filter name already in use"
```

Reply:
Yep sorry I've forgot to change the message in the Exception I want to keep "used" :D

Reply:
I've changed the message in the Exception